editors/sc-ide/core/main.cpp

```diff
@@ -131,6 +131,7 @@ int main( int argc, char *argv[] )
 void SingleInstanceGuard::onNewIpcConnection()
 {
     mIpcSocket = mIpcServer->nextPendingConnection();
+	mIpcChannel->setSocket(mIpcSocket);
```

Comment:
Sorry, I was using another computer than the one I usually use, and it was configured.